eip/EIPS/eip-4337.md

```diff
@@ -391,78 +451,104 @@ The result `SHOULD` be set to true if and only if the request passed simulation
     entryPoint // address
   ]
 }
-
-# Response
+```
+Response:
+```
 {
   "jsonrpc": "2.0",
   "id": 1,
   "result": true
 }
 ```
-
-`eth_supportedEntryPoints`
-
-eth_supportedEntryPoints returns an array of the entryPoint addresses supported by the client. The first element of the array `SHOULD` be the entryPoint addressed preferred by the client.
-
-```json=
-# Request
+#### Example failure responses:
+```json
 {
   "jsonrpc": "2.0",
   "id": 1,
-  "method": "eth_supportedEntryPoints",
-  "params": []
+  "error": {
+    "message": "Wallet didn't pay prefund",
+    "code": -33000
```

Comment:
Codes don't match. Just saying.

Reply:
Right.  Is this one a -32500 (failed simulation)?  Or a separate code that we don't currently have?